Dockerfile

```
@@ -36,6 +36,7 @@ LABEL Description="This is a base image, which provides the Jenkins agent execut
 
 ARG AGENT_WORKDIR=/home/${user}/agent
 
+RUN echo 'deb http://deb.debian.org/debian stretch-backports main' > /etc/apt/sources.list.d/stretch-backports.list
 RUN apt-get update && apt-get install git-lfs
```

I am surprised that this works, since the [Debian documentation](https://backports.debian.org/Instructions/#index3h2) says that:

> All backports are deactivated by default (i.e. the packages are pinned to 100 by using ButAutomaticUpgrades: yes in the Release files. If you want to install something from backports run:
> 
> `apt-get -t stretch-backports install "package"`

I would suggest (not mandatory, just a suggestion) that the command should use the syntax recommended by the Debian documentation.

```suggestion
 RUN apt-get update && apt-get install -t stretch-backports git-lfs
```

I assume it works "as-is" because there is no `git-lfs` package in the base Debian distribution and that causes the stretch-backports repository to be selected as the only repository that provides a `git-lfs` package.

---

Neither the `apt-get install` syntax nor the git version was a deliberate choice. I’m definitely fine with following the Debian docs.

> I assume it works "as-is" because there is no `git-lfs` package in the base Debian distribution and that causes the stretch-backports repository to be selected as the only repository that provides a `git-lfs` package.

Sounds reasonable; I don’t know for sure either.